File: Beginner/FLOW004.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main(){
	int t;
	cin >> t;
	long n;
	
	for(int i=0;i<t;i++){
		cin >> n;
		
		int last_digit = n%10;
		int first_digit;
		long rem = n;
		long tens = 1000000; 
		while(n == rem){
			first_digit = n/tens;
			rem = n%tens;
			tens /= 10;
		}
		int output = first_digit+last_digit;
		cout << output << endl;
	}
	
	return 0;
}

File: Beginner/FLOW004.java
import java.util.*;
import java.lang.*;
import java.io.*;

class CodeChef{
	
	public static void main(String[] args){
		
		Scanner scanner = new Scanner(System.in);
		
		int t = scanner.nextInt();
		scanner.nextLine();
		String n;
		
		for(int i=0;i<t;i++){
			n = scanner.nextLine();
			int len = n.length();
			int last_digit = Integer.parseInt(n)%10;
			int tens = (int)Math.pow(10,(len-1));
			int first_digit = Integer.parseInt(n)/(tens);
			System.out.println(first_digit + last_digit);
		}
	}
}